### ISSUE TITLE
Not able to interact with elements in new window

### ISSUE THREAD

**Issue by saikrishna321** [getgauge/taiko#31]
### Expected behavior
Should be able to interact with elements on new window

### Actual behavior
Not able to find elements on new window

### Steps to reproduce
1. openBrowser()
2. goto('https://getgauge.io')
3. click(text('1194'))

### Versions
```
Version: 0.0.9 (HeadlessChrome/64.0.3264.0)
Node version:  v8.10.0
```

**Comment by NivedhaSenthil** [comment]
Seems like the text "1194" has been changed. I am able to click on "1196" (Github start count when I tried)
Taiko in master is unstable might have bugs, it is being rewritten using Chrome Remote Interface instead of puppeteer.

**Comment by NivedhaSenthil** [comment]
Yes, Taiko should be able to interact with elements in new window.

**Comment by saikrishna321** [comment]
> Taiko should be able to interact with elements in new window

Nope i tried and it didn't work.

**Comment by NivedhaSenthil** [comment]
Can you please give details on what you had tried and what failure occurred ? 
Will help us fix if there is any issue.

**Comment by saikrishna321** [comment]
@NivedhaSenthil sorry for delay response, I missed your comment.

I'm running taiko from the master. How do I provide failure stack trace?

**Comment by NivedhaSenthil** [comment]
Just give us the script you had tried or if you tried from REPL generate script using .code. And what should be the excepted behaviour of the scenario.

**Comment by saikrishna321** [comment]
> goto('https://getgauge.io')
 ✔ Navigated to url "https://gauge.org/index.html"
> openBrowser()
 ✔ Browser and page initialized
> openBrowser()
 ✔ Browser and page initialized
> openBrowser()
 ✔ Browser and page initialized
> goto('https://getgauge.io')
 ✔ Navigated to url "https://gauge.org/index.html"
> click(text('1244'))
 ✔ Clicked element with text "1244"

this would open a new tab with https://github.com/getgauge/gauge and i was trying to interact with elements on the new tab 

> click(text('Sign in'))
 ✘ Error: Element not found, run `.trace` for more info.

**Comment by NivedhaSenthil** [comment]
Was able to reproduce with below script

```
(async () => {
    try {
        await openBrowser();
        await goto('https://getgauge.io');
        await click(link(toLeftOf('Plugins')));
        await highlight(link("Features"));
    } catch (e) {
        console.error(e);
    } finally {
        closeBrowser();
    }
})();
```

Expected to highlight link with text 'Features' on the new window which did not happen and gives 
`✘ Error: Link with text Features not found`

wherein if we goto the page directly things works fine

```
await goto("https://github.com/getgauge/gauge");
await highlight(link("Features"));
```

**Comment by NivedhaSenthil** [comment]
Fix should be available in [commit](https://github.com/getgauge/taiko/commit/8a33ac23f7b857ecfef26350e3506bb152261d1b). 

Below scripts demos on accessing elements in newly opened page, switching tab and accessing elements there.
```
(async () => {
    try {
        await openBrowser();
        await goto('https://gauge.org');
        await click(link(toLeftOf('Plugins')));
        await highlight(link("Features"));
        await switchTo('Open Source Test Automation Framework | Gauge'); 
        await click("Plugins");
    } catch (e) {
        console.error(e);
    } finally {
        closeBrowser();
    }
})();
```

**Comment by NivedhaSenthil** [comment]
Example that shows how the above script works.

![Example that shows how the above script runs](https://user-images.githubusercontent.com/6358540/44071616-df72d420-9fa7-11e8-90ef-6a697df13ef4.gif)

**Comment by tsachit** [comment]
click is still not working after switchTo on pop up window

**Comment by zabil** [comment]
What version of taiko are you using?

**Comment by tsachit** [comment]
> What version of taiko are you using?

I'm sorry, I didn't realize taiko had new version. I was using Version: 1.0.3. 
Just upgraded to Version: 1.0.6, now i don't even have to use switchTo to use popup window. That's amazing. Thanks
